packages/amplify_core/lib/amplify_core.dart

```diff
@@ -28,6 +30,7 @@ export 'src/config/notifications/notifications_config.dart';
 export 'src/config/storage/storage_config.dart';
 
 /// HTTP
+export 'src/http/amplify_category_method.dart';
```

Comment:
issue: amplify_category_method.dart is an internal library so instead of exporting it from here use ```// ignore: invalid_use_of_internal_member``` in cloudwatch_logger_plugin.dart
```
// ignore: invalid_use_of_internal_member
import 'package:amplify_core/src/http/amplify_category_method.dart';
```